components/02-molecules/meta/basic-meta/_yds-basic-meta.scss

```diff
@@ -17,7 +17,12 @@
     > *:not(:first-child) {
       margin-left: var(--size-spacing-5);
       padding-left: var(--size-spacing-5);
-      border-left: var(--border-thickness-1) solid;
+    }
+
+    .date-time::before {
+      position: relative;
+      content: '|';
+      right: var(--size-spacing-5);
```

Comment:
Similar to above, you have this only applying to the `data-time` element in the meta, and we actually want it to apply to _anything_ in the meta, so this can be moved up inside the previous `> *:not(:first-child)` section.

Also, the positioning isn't quite right. If you inspect it, you'll see it's off center. There are a few things we need to do to make it exactly centered.

- [ ] Let's add `display: flex` to the `basic-meta` class, so that there's no extra space between these inline elements.
- [ ] Set `position: relative` on the items (above the `margin-left` and `padding-left` stuff)
- [ ] Set `position: absolute` on the `before` element
- [ ] You'll notice the `right` positioning you have here isn't actually doing anything in the browser (there's a technical limitation of custom properties that makes them not available to pseudo-elements like `::before`), and since this is content it's getting a little extra space on the left, so let's used a specific value - in this case I think `left: -0.225rem;` will make it align the way we want.